src/Common/Admin/Settings_Sidebar.php

```diff
@@ -42,64 +33,77 @@ class Settings_Sidebar extends Section {
 	public function render() {
 		?>
 		<div class="tec-settings-form__sidebar">
-		<?php do_action( 'tec_settings_sidebar_start' ); ?>
-			<div class="tec-settings-form__sidebar-section tec-settings-form__sidebar-header">
-				<?php do_action( 'tec_settings_sidebar_header_start' ); ?>
-				<?php
-				$this->render_header_image();
-				$this->render_title();
-				?>
-				<?php do_action( 'tec_settings_sidebar_header_end' ); ?>
-			</div>
-
-			<?php foreach ( $this->sections as $section ) : ?>
+			<?php do_action( 'tec_settings_sidebar_start', $this ); ?>
+			<?php foreach ( $this->get_sections() as $section ) : ?>
 				<div class="tec-settings-form__sidebar-section">
 					<?php $section->render(); ?>
 				</div>
 			<?php endforeach; ?>
-			<?php do_action( 'tec_settings_sidebar_end' ); ?>
+			<?php do_action( 'tec_settings_sidebar_end', $this ); ?>
 		</div>
 		<?php
 	}
 
 	/**
-	 * Set the header image for the sidebar.
+	 * Alias to prepending a section to the sidebar.
 	 *
 	 * @since 6.1.0
 	 *
-	 * @param Image $image The image to set.
+	 * @param Section $section The section to add.
 	 *
 	 * @return void
 	 */
-	public function set_header_image( Image $image ) {
-		$this->header_image = $image;
+	public function add_section( Section $section ): self {
+		return $this->append_section( $section );
 	}
 
 	/**
-	 * Add a section to the sidebar.
+	 * Add a section to the end of the sidebar array of sections
 	 *
-	 * @since 6.1.0
+	 * @since TBD
 	 *
 	 * @param Section $section The section to add.
 	 *
 	 * @return void
 	 */
-	public function add_section( Section $section ) {
+	public function append_section( Section $section ): self {
```

Comment:
To add consistency with `prepend_` method.

Reply:
> I think that this gives us the opportunity to add a second param to `add_section`

Respectfully, I disagree. I think we should have less duplicity in our methods rather than more. It may be small, but it still introduces unnecessary complexity into this method.